views/header/header-left.jade

```diff
@@ -3,3 +3,5 @@ ul(ng-controller="HeaderLeftCtrl as headerLeftCtrl", class="nav navbar-nav")
     a(href="/") Food Recalls
   li(class="{{getCurrentPath() == '/about' && 'active' || ''}}")
     a(href="/about") About
+  li(class="{{getCurrentPath() == '/contact' && 'active' || ''}}")
```

Comment:
Yeah, I realized as soon as I read your comment that I was hiding the content haha.